### PR TITLE
[python] Integrate test run strategy change

### LINKED ISSUE #10488: [Bug] [python] integrate test run strategy change
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

https://github.com/apache/dolphinscheduler/pull/10468#discussion_r899801771 some service or mapper change will failed python  api and it will not trigger the python integrate test, we have to change the strategy  CI of python api run

### What you expected to happen

ATT

### How to reproduce

ATT

### Anything else

ATT

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhongjiajie**
this patch change python integrate test strategy,
before that, only content change in dir
dolphinscheduler-python will run the test. And it
will fail our python api startter when some code
change in java gateway. So we should start the
integrate test except docs change.

and in the future, we may add python ci to
.asf.yaml to make sure it pass

close: #10488

### CODE REVIEW

**Review by SbloodyS — DISMISSED** [review]
LGTM

**Comment by zhongjiajie** [comment]
do not merge until #10792 merged

**Comment by zhongjiajie** [comment]
> do not merge until #10792 merged

#10792 already merged, restart this PR

**Review by SbloodyS — DISMISSED** [review]

**Review by SbloodyS — APPROVED** [review]

**Pull request merged**
